### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by jaswantcoder**
*Issue # updated documentation*

### CODE REVIEW

File: EXAMPLE.md
Comment (debanshuk) [inline EXAMPLE.md]: Nit: We can skip this part from the example, as we don't want to promote usage of IAM users.

File: EXAMPLE.md
Comment (debanshuk) [inline EXAMPLE.md:30]: At some place in this file we should mention that 'KMS_SIGNING_KEY_ID' (and other KMS key IDs) can be key-id, key-id-arn, key-alias or key-alias-arn, based upon the user's needs.

File: EXAMPLE.md
Comment (debanshuk) [inline EXAMPLE.md]: Nit: 'kms' -> 'KMS'.

**Comment by mjsheikh** [comment]
Please enable atleast the following branch protection rules for the `main` branch with the following settings:
1. Require a pull request before merging
a. Require approvals
b. Dismiss stale pull request approvals when new commits are pushed

File: EXAMPLE.md
Comment (jaswantcoder) [reply EXAMPLE.md]: removed

File: EXAMPLE.md
Comment (jaswantcoder) [reply EXAMPLE.md:30]: added a Note at the end of file

File: EXAMPLE.md
Comment (jaswantcoder) [reply EXAMPLE.md]: done

**Review by mjsheikh — APPROVED** [review]

**Review by debanshuk — COMMENTED** [review]
It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

**Comment by jaswantcoder** [comment]
> It seems there a 4 commits in this PR with same/similar commit messages. Could you please merge these commits to a single commit?

Yes, squashed commits into one and merged. Deleted branch

**Pull request merged**
